src/Runtime/OMResize.inc

```diff
@@ -306,10 +306,13 @@ static void interpolate_nd_OMTensor(OMTensor *output_OMT, OMTensor *data,
   free(coeffs_buffer);
 }
 
+// The parameters that are not used are commented out.
+// I, Tong Chen, forgot why only the default value for them are used.
+// ToFix: add the support for other value if needed
 void Resize_Scales(
     OMTensor *output, OMTensor *data, OMTensor *scales,
-    char *coordinate_transformation_mode_str, float coeff_a,
-    int exclude_outside, float extrapolation_value, char *mode_str,
+    /* char *coordinate_transformation_mode_str, float coeff_a,*/
+    /* int exclude_outside, float extrapolation_value, */ char *mode_str,
```

Comment:
Would you be able to investigate with Blame who added them originally?

Reply:
I should be blamed for these code. The previous implementation was rewritten from a python implementation. I inherited the interface but not all the implementation.

Reply:
so should we just remove them?